charters/wot-wg-charter-draft-2019.html

```diff
@@ -178,39 +186,369 @@ <h2>Scope</h2>
         <p><i class="todo">Brief background of landscape, technology, and relationship to the Web, users, 
         developers, implementers, and industry.</i></p>
         -->
-        <p>The Working Group has X items in scope:</p>
+        <p>The Working Group has the following items in scope:</p>
         
         <div class="summary">
           <p style="text-align:left;font-style:italic">Scope Summary</p>
           
           <dl>
-            <dt><strong>Item 1:</strong></dt>
-            <dd>summary of item 1.</dd>
-            <dt><strong>Item 2:</strong></dt>
-            <dd>summary of item 2.</dd>
+            <dt><strong>TD Security Schemes:</strong></dt>
+            <dd>Vocabulary for new security schemes supporting targetted protocols and use cases.</dd>
+            <dt><strong>TD Link Relation Types:</strong></dt>
+            <dd>Definition of specific link relation types for specific relationships.</dd>
+	    <dt><strong>Architecture Use Cases and Vocabulary</strong></dt>
+            <dd>Definition of new use cases, architectural patterns, and concepts.</dd>
+            <dt><strong>TD Vocabulary:</strong></dt>
+            <dd>Extensions to TD vocabulary definitions.</dd> 
+            <dt><strong>Protocol Vocabulary:</strong></dt>
+            <dd>Extensions to protocol vocabulary definitions and protocol bindings.</dd> 
+            <dt><strong>Smart Home Profile:</strong></dt>
```

Comment:
We should not pick a specific application domain for the definition of a profile.
If we contrain the profile to the smart home, we will shy away the industrial players.

Suggest to generalize it to:
- define a common profiling mechanism
- define profiles for specific application domains and use cases

Reply:
Your generalization is fine.  Eventually we need to resolve the vertical/horizontal issues but this can be done during WG discussion, it does not need to be done while writing the charter.

Reply:
updated to remove emphasis on Smart Home and generalize.